docs/compression_algorithms/Pruning.md

```diff
@@ -27,6 +27,74 @@ Where ![L_j](https://latex.codecogs.com/png.latex?L_j) is j-th convolutional lay
 
 Then during pruning filters with smaller ![G(F_i)](https://latex.codecogs.com/png.latex?G%28F_i%29) importance function will be pruned first.
 
+#### Algorithm statistics
+A model compression can be measured by two main metrics: filter pruning level and FLOPs pruning level. While 
+filter pruning level shows the ratio of removed filters to the total number of filters in the model, FLOPs pruning level 
+indicates how the removed filters affect the number of floating point operations required to run a model. 
+
+During the algorithm execution several compression statistics are available:
+```
+Statistics of the pruned model:
++---------+-------+---------+---------------+
+|    #    | Full  | Current | Pruning level |
++=========+=======+=========+===============+
+| GFLOPS  | 0.602 | 0.241   | 0.599         |
++---------+-------+---------+---------------+
+| MParams | 3.470 | 1.997   | 0.424         |
++---------+-------+---------+---------------+
+| Filters | 17056 | 10216   | 0.401         |
++---------+-------+---------+---------------+
+Prompt: statistic pruning level = 1 - statistic current / statistic full.
+Statistics of the filter pruning algorithm:
++---------------------------------------+-------+
+|           Statistic's name            | Value |
++=======================================+=======+
+| Filter pruning level in current epoch | 0.500 |
++---------------------------------------+-------+
+| Target filter pruning level           | 0.800 |
++---------------------------------------+-------+
+```
+
+##### Model statistics
```

Comment:
```suggestion
##### Model statistics
Columns `Full` and `Current` represents the value of correspondent statistics in a original model and in a compressed one in current state accordingly.
```

Reply:
How about to describe columns first, and then describe all the rows?

Reply:
Done